airbyte-integrations/connectors/source-amazon-ads/source_amazon_ads/streams/report_streams/report_streams.py

```diff
@@ -250,7 +250,7 @@ def _send_http_request(self, url: str, profile_id: int, json: dict = None, is_do
             session = self._report_download_session if is_download_report else self._session
             response = session.get(url, headers=headers)
         if response.status_code == HTTPStatus.TOO_MANY_REQUESTS:
-            raise TooManyRequests()
+            raise TooManyRequests("429: Too many requests during report creation. Please try again later...")
```

Comment:
Is it planned to make this [AirbyteTracedException](https://airbytehq-team.slack.com/archives/C02U9R3AF37/p1730500230549159?thread_ts=1730494293.143359&cid=C02U9R3AF37)?

> instead of emitting a custom error exception like TooManyRequests, we should be emitting an AirbyteTracedException with the error type as Transient and the appropriate message

Reply:
Type `transient_error` would be awesome!

Reply:
updated!